packages/material-icon/README.md

```diff
@@ -0,0 +1,25 @@
+# Material Icon React
+
+MDC React Material Icon is a component for material icons.
+
+## Installation
+
+```
+npm install @material/react-material-icon
+```
+
+## Usage
+
+### JSX Structure
```

Comment:
is this JSX structure section even necessary?

Reply:
In this scenario no. But with Cards I imagine we may have multiple components like:

```html
<Cards>
  <Card />
  <Card />
</Cards>
```

Should we omit it in this case?